web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/behandling/aksjonspunkt/AksjonspunktTjeneste.java

```diff
@@ -355,8 +354,8 @@ private void bekreftAksjonspunkt(BehandlingskontrollKontekst kontekst,
         overhoppResultat.leggTil(delresultat);
         byggVilkårResultat(vilkårBuilder, delresultat);
 
-        if (delresultat.kreverTotrinnsKontroll() && aksjonspunktStøtterTotrinn(aksjonspunkt)) {
```

Comment:
OBS endringer siden det ikke var støttet at et AP først gikk fra å være totrinn til å ikke bli totrinn. Vil dette gi problemer andre steder? @jolarsen ?

Reply:
For å støtte caset 

1. Saksbehandler overstyrer brev
2. Beslutter sender tilbake
3. Saksbehandler sletter overstyrt brev
4. Brevet går ikke til totrinn siden det er slettet

Reply:
Blir så få tilfeller uansett, så regner med det er greit at de må godkjenne en ekstra